README.md

```diff
@@ -68,8 +68,9 @@ yarn test-with-node
 
 Run a single test module with `yarn test <module name or prefix>`.
 
-## Lint
-`yarn lint --fix`
+## Linting and type checking
+We use JSDoc and TypeScript for static type checking during the CI process. PRs must pass linting and type checking to be accepted.
+You can run via: `yarn lint --fix` and `yarn type-check`
```

Comment:
Would `type-check` fix the lint errors as well?

Reply:
nope, they have to be ran separately. we could concat into one command but then hard to see exactly where things go wrong in ci